### DISCUSSION TITLE
Why does it turn into a copy action after I select the text?

### DISCUSSION THREAD

**Discussion by fxzxmicah** [hyprwm/Hyprland#8387] [Q&A]
I don't know if it's a hyprland problem, but it can only be a hyprland problem because the same problem occurs in all software.

**Comment by vaxerski** [comment]
you mean drag? We send the first available action iirc, if copy is supported we do copy

**Reply by fxzxmicah** [reply]
I don't know if we're talking about the same thing.
But now as soon as I select the text with the mouse, it will automatically copy the text, so that I can't use it normally.

**Comment by fufexan** [comment]
https://wiki.hyprland.org/FAQ/#how-to-disable-middle-click-paste

**Reply by fxzxmicah** [reply]
It's not that, but I've found the reason. It's my vnc client.
